src/pages/Checkout/Checkout.js

```diff
@@ -0,0 +1,69 @@
+import { useLocation, useNavigate } from "react-router-dom";
+
+import { Header } from "../../components";
+
+import styles from "./Checkout.module.css";
+
+const formatDate = (dateParam) => {
+  const date = new Date(dateParam);
+  const day = date.getDate();
+  const month = date.getMonth() + 1;
+  const year = date.getFullYear();
+
+  return `${day < 10 ? '0' + day : day}/${month < 10 ? '0' + month : month}/${year}`;
+};
+
+export default function Checkout() {
+  const { state: menuData } = useLocation();
+  const navigate = useNavigate();
+
+  const handleConfirm = () => {
+    alert('Confirmando');
+    // TODO: BACK
+    navigate(-1)
+  }
```

Comment:
Ao testar, vi que o botão dispara o alert duas vezes. Usando um useCallback vc evita que esse componente seja re-renderizado e evita problemas.
```suggestion
  const handleConfirm = useCallback(() => {
    alert('Confirmando');
    // TODO: BACK
    navigate(-1)
  }, []);
```